networkconfig/holesky-stage.go

```diff
@@ -16,7 +16,7 @@ var HoleskyStage = NetworkConfig{
 	GenesisEpoch:         1,
 	RegistrySyncOffset:   new(big.Int).SetInt64(84599),
 	RegistryContractAddr: "0x0d33801785340072C452b994496B19f196b7eE15",
-	AlanForkEpoch:        99999999,
+	AlanForkEpoch:        78286,
```

Comment:
do we want to move this away from pre-fork on stage?

Reply:
This is just a temporary change so I can do some tests in stage 😅 
I'll remove it after the testing is done

Reply:
A note for everyone for future testing. Instead of editing your branch - create a new branch called `test/youbranchname` or `deploy/youbranchname` and deploy it instead polluting your branch with irrelevant changes you'll need to revert..

Reply:
Got it! Thanks